tools/db/loadTransientDBViews.sh

```diff
@@ -1,105 +0,0 @@
-#!/bin/bash
-
-#
-# Copyright 2015-2016 IBM Corporation
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-#
-# drop and recreate the transient whisk cloudant views.
-# NOTE: before editing this file, review the notes in
-# whisk.core.entity.WhiskStore as any changes here to the views
-# may require changes to the supporting query methods.
-#
-
-SCRIPTDIR="$(cd $(dirname "$0")/ && pwd)"
-source "$SCRIPTDIR/common.sh"
-
-function addRevision() {
```

Comment:
are we retaining the ability to reload the views without wiping the db (as this script allowed)?